brands/ghostery/branding/pref/firefox-branding.js

```diff
@@ -17,157 +19,116 @@ pref("app.update.url.manual", "https://www.ghostery.com/download/");
-pref("general.useragent.compatMode.firefox", true);
-
-// Recommend extensions/features as you browse
-pref("browser.newtabpage.activity-stream.asrouter.userprefs.cfr.features", false);
+pref("extensions.fxmonitor.enabled", false); // Firefox Monitor
+pref("signon.management.page.breach-alerts.enabled", false); // Firefox Lockwise
+pref("signon.management.page.breachAlertUrl", "");
+// PREF: Disable Extension Recommendations (CFR: "Contextual Feature Recommender")
+// [1] https://support.mozilla.org/en-US/kb/extension-recommendations
 pref("browser.newtabpage.activity-stream.asrouter.userprefs.cfr.addons", false);
+pref("browser.newtabpage.activity-stream.asrouter.userprefs.cfr.features", false);
+pref("extensions.htmlaboutaddons.recommendations.enabled", false);
+pref("extensions.getAddons.showPane", false);
 
-// Remove addons.mozilla.org from set of domains that extensions cannot access
-pref("extensions.webextensions.restrictedDomains", "accounts-static.cdn.mozilla.net,accounts.firefox.com,addons.cdn.mozilla.net,api.accounts.firefox.com,content.cdn.mozilla.net,discovery.addons.mozilla.org,install.mozilla.org,oauth.accounts.firefox.com,profile.accounts.firefox.com,support.mozilla.org,sync.services.mozilla.com");
-
-// Support and feedback URLs
-pref("app.support.baseURL", "https://get.ghosterybrowser.com/support/");
-pref("app.feedback.baseURL", "https://www.ghostery.com/support/");
-
-// Override settings server to our own one
-pref("services.settings.server", "https://get.ghosterybrowser.com/settings/v1");
-
-// Disable extension abuse reporting (#186)
-pref("extensions.abuseReport.enabled", false);
-
-// Disable about:welcome page
+/** HOMEPAGE ***/
+pref("startup.homepage_override_url", "");
+pref("startup.homepage_welcome_url", "");
+pref("startup.homepage_welcome_url.additional", "");
 pref("browser.aboutwelcome.enabled", false);
 
-// Disable activity stream (#131)
+/** NEW TAB PAGE & ACTIVITY STREAM ***/
+pref("browser.startup.page", 3);
 pref("browser.library.activity-stream.enabled", false);
 pref("browser.newtabpage.activity-stream.discoverystream.enabled", false);
-
-// Disable corroborator (#141)
-pref("corroborator.enabled", false);
-
-// Disable password breach alerts
-pref("signon.management.page.breach-alerts.enabled", false);
-
-// Disable Firefox VPN promo
-pref("browser.privatebrowsing.vpnpromourl", "");
-
-// Disable dropdown suggestions with empty query
-pref("browser.urlbar.suggest.topsites", false);
-
-// Fetch shavar updates from Ghostery endpoint
-pref("browser.safebrowsing.provider.mozilla.updateURL", "https://get.ghosterybrowser.com/shavar/downloads?client=SAFEBROWSING_ID&pver=2.2");
-
-// Firefox Monitor
-pref("extensions.fxmonitor.enabled", false);
-
-// Enable HTTPS only mode (#367)
-pref("dom.security.https_only_mode", true);
-
-// Microphone and camera kill switch (#370)
-pref("privacy.webrtc.globalMuteToggles", true);
```

Comment:
please bring it back

Reply:
HTTPS-only prefs are [here](https://github.com/ghostery/user-agent-desktop/blob/19706e3396cf9a3eea8255ef98409bf4602eeddf/brands/ghostery/branding/pref/Better-Fox.js#L85-L89) in Betterfox.

`pref("privacy.webrtc.globalMuteToggles", true);` was removed because it is still considered experimental in Firefox Nightly. As a principle, I excluded Nightly experimental prefs. Would you like to include it anyway in BetterFox?

Reply:
Yeah, we are pushing this one to our users. Works good so far.

Reply:
Yes, sir! I will add this to `Better-Fox.js`. :)